test/routes/heartbeats.routes.test.js

```diff
@@ -0,0 +1,53 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import * as apiUtil from '../test-utils/api.util';
+import server from '../globalSetupTeardown.test';
+
+chai.use(chaiHttp);
+const should = chai.should();
+const expect = chai.expect;
+const endpoint = '/heartbeats';
+
+describe('Testing Tokens Routes', () => {
+  describe(`POST /heartbeats`, () => {
+    it('should return a 400 if the user is does not have a projectId', (done) => {
```

Comment:
Does not have permission for this projectId?

Reply:
Is there even a user specified? I think this description is wrong

Reply:
The user meaning the person make the request. 

But I agree, the verbiage is not good.